packages/components/src/components/textarea/textarea.examples.md

```diff
@@ -0,0 +1,183 @@
+
+## Controlled
+
+In the controlled approach, the `p-textarea` value is externally managed. While the internal value will be updated
+automatically, you can use the `input`, `change` or `blur` event to update the external state.
+
+<Playground :frameworkMarkup="controlledExample" :config="{ ...config, withoutDemo: true }">
+<p-textarea name="some-name" label="Some Label" :theme="theme" @input="updateControlledExample"></p-textarea>
+<br>
+<p-text :theme="theme">{{ selectedValueControlled }}</p-text>
+</Playground>
+
+## With description text
+
+A description text can be added to explain the meaning of a specific form field. It's meant to be a textual enhancement
+of the label text and is technically connected with the `hide-label` property.
+
+<Playground :markup="withDescriptionText" :config="config"></Playground>
+
+### <A11yIcon></A11yIcon> Accessibility hints
+
+In general, you should **prevent** using the `disabled="true"` state. Disabled elements are not reachable (focusable)
+anymore and can be missed by screen reader users. They can be confusing for sighted users as well by not pointing out
+why these elements are disabled. A good practice when to use the disabled state is during **form submission** to prevent
+changes while this process is performed.
+
+## Counter
+
+If the `maxlength` attribute is present on the `p-textarea` element, a counter will be displayed in the corner.  
+To hide it you can set `showCounter` to `false`.
+
+<Playground :markup="counter" :config="config"></Playground>
+
+## Validation states
+
+The `p-textarea` component supports the visualisation of inline validation.
+
+<Playground :markup="stateMarkup" :config="config">
+  <PlaygroundSelect v-model="state" :values="states" name="state"></PlaygroundSelect>
+</Playground>
+
+## Slots
+
+Sometimes it's useful to be able to render markup (e.g. an anchor tag) for `label` or `message`. Therefore a named slot
+can be used. Make sure **not** to define the corresponding property on the host element when a named slot is used
+(because a property definition is preferred over a named slot). For named slots only
+[phrasing content](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Content_categories#Phrasing_content) is
+allowed.
+
+<Playground :markup="slots" :config="config"></Playground>
+
+### <A11yIcon></A11yIcon> Accessibility hints
+
+If using **slotted contents** to serve form elements, make sure to provide the right **ARIA attributes** to give screen
+reader users the corresponding information:
+
+1. Add a unique ID to the `slot="label"` element
+1. Add a unique ID to the `slot="message"` element (if they are created)
+1. Add corresponding `aria-labelledby="some-label-id"` to the `textarea` element which points to the `label` ID
+1. Add corresponding `aria-describedby="some-description-id some-message-id"` to the `textarea` element which points to
+   both, the `description` ID (if set) and the `message` ID when the (error/success) message appears
+
```

Comment:
All this A11y hints can also be removed.

Reply:
removed